controllers/fenceagentsremediation_controller_test.go

```diff
@@ -209,16 +211,77 @@ var _ = Describe("FAR Controller", func() {
 			})
 		})
 	})
+	Context("Reconcile with OutOfServiceTaint strategy", func() {
+		nodeKey := client.ObjectKey{Name: workerNode}
+		farNamespacedName := client.ObjectKey{Name: workerNode, Namespace: defaultNamespace}
+		farNoExecuteTaint := utils.CreateFARNoExecuteTaint()
+		outOfServiceTaint := utils.CreateOutOfServiceTaint()
+		conditionStatusPointer := func(status metav1.ConditionStatus) *metav1.ConditionStatus { return &status }
+		BeforeEach(func() {
+			farPod := createRunningPod("far-manager-test", farPodName, "")
+			DeferCleanup(deletePod, farPod)
+		})
+		JustBeforeEach(func() {
+			// Create node, and at the end clean it up with DeferCleanup
+			Expect(k8sClient.Create(context.Background(), node)).To(Succeed())
+			DeferCleanup(k8sClient.Delete, context.Background(), node)
+		})
+
+		When("creating valid FAR CR", func() {
+			var va *storagev1.VolumeAttachment
```

Comment:
I think everything about VAs can be removed, see https://github.com/medik8s/fence-agents-remediation/pull/102

Reply:
Same here. I will remove this change.